sfs/array.py

```diff
@@ -484,6 +560,15 @@ def weights_midpoint(positions, closed):
     (N,) numpy.ndarray
         Weights of secondary sources.
 
+    Examples
+    --------
+    .. plot::
+        :context: close-figs
+
+        x0, n0, a0 = sfs.array.circular(2**8, 1)
+        a = sfs.array.weights_midpoint(x0, closed='True')
+        print(max(abs(a0-a)))  # check error against analytic solution, i.e. 6.160347305397829e-07
```

Comment:
The output of `print()` is not shown when using the plot directive.

You could use doctests, like e.g. in http://python.sfstoolbox.org/en/0.4.0/utilities.html#sfs.util.as_delayed_signal

Also, the line is too long.

Reply:
nice tool, thanks! changed to this handling